tests/acceptance/SubscriptionContext.php

```diff
@@ -252,4 +263,13 @@ public function iChangeTheSubscriptionAll()
             ->update($this->subscription);
         $this->iQueryForTheSubscription();
     }
+
+    /**
+     * @Then the subscription customer id must be the same as the retrieved
+     */
+    public function theSubscriptionCustomerIdMustBeTheSameAsTheRetrieved()
+    {
+        $subscriptionCustomer = $this->subscription->getCustomer();
+        assertEquals($subscriptionCustomer->getId(), $this->customer->getId());
```

Comment:
Asserting the response's customer_id is equal to the request one doesn't really test whether the parameter was sent or not. It could just mean the api is checking if a customer with the provided document_number/email exists.
In this case, a unit test works best.

Reply:
Sure, i just added them =D

Pls review again @henriquekano

Reply:
@leonardo-barbosa you added your code in the `getPayload` method, why are you testing the `getCustomer` method? I didn't understand

Reply:
Hey @piiih , my thought was to make sure that the customer id i sent to the API is the same returned in the response.